common/static/common/js/spec/main_requirejs.js

```diff
@@ -153,6 +155,10 @@
         }
     });
 
+    // None of the common source files require jquery-migrate
+    // (it is loaded by LMS via base_vendor_js, and CMS by common_libraries).
+    require(['jquery-migrate']);
```

Comment:
Other libraries can easily be added--

require(['jquery-migrate', 'and-another-library']);